### ISSUE TITLE
why b4-backbone's training loss is  larger than b0?

### ISSUE THREAD

**Issue by wangmingrun1314** [zylo117/Yet-Another-EfficientDet-Pytorch#559]
Hi  Master.
    I have studied this pro for someday. the most  problem that drew my attention is the b4-arc didn't have a lower loss(compared with bo) when trained on   my  own dataset.i don't know whether the anchor position/size  influences training process.But reasonably,b4 should performs better than b0.
    Addittionally,if the anchor position center setting like this:
                                             pyramid1:1 3 5 7 9 11 13 15 ....
                                             pyramid2:3 7 11 15....
                                             pyramid3:7  15 ....
    is more suitable under Efficientnet framework.

**Comment by zylo117** [comment]
Deeper network always has larger loss at first or at the end, if underfitting.
